# Make sure the tool contains the config.xml file (contains-config)

This rule is important to check the structure of your tool.  Make sure to include the config.xml in every too.


## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js

configuration.xml

```

Examples of **correct** code for this rule:

```js

config.xml

```


## When Not To Use It

This rule must remain on at all times

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
